exec/cnex/lib/time_posix.c

```diff
@@ -0,0 +1,39 @@
+#define _POSIX_C_SOURCE 200809L
+#include <stdlib.h>
+#include <sys/time.h>
+#include <time.h>
+#include <unistd.h>
+
+#include "cell.h"
+#include "exec.h"
+#include "number.h"
+#include "stack.h"
+
+
+void time_sleep(TExecutor *exec)
+{
+    // Note: The number on the top of the stack is in SECONDS!
+    uint64_t ms = number_to_uint64(number_multiply(top(exec->stack)->number, number_from_uint32(1000))); pop(exec->stack);
```

Comment:
This throws away anything less than one millisecond. Why do this? It would be better to preserve the full precision since nanosleep() supports nanosecond resolution.

Reply:
Fixed so that we can actually sleep by nanoseconds now.